languages/go/example/example.go

```diff
@@ -19,11 +19,15 @@ func main() {
 	organizationIDStr := os.Getenv("ORGANIZATION_ID")
 	projectName := os.Getenv("PROJECT_NAME")
 
+	// Configuring the statePath is optional, pass nil
+	// in AccessTokenLogni() to not use state
```

Comment:
```suggestion
	// in AccessTokenLogin() to not use state
```

Reply:
Oh, that's terrible 🤦‍♂️ Fixed, sorry about that!